skype/skype-ps/skype/Get-CsUserAcp.md

```diff
@@ -37,6 +37,8 @@ Get-CsUserAcp [[-Identity] <UserIdParameter>] [-Credential <PSCredential>] [-Fil
 ```
 
 ## DESCRIPTION
+**Deprecation Notice:** Integration of Skype for Business with third-party audio conferencing providers has been retired. More details are here - https://docs.microsoft.com/en-us/skypeforbusiness/legal-and-regulatory/end-of-integration-with-3rd-party-providers. As a result, cmdlet no longer would work.
```

Comment:
```suggestion
Deprecation Notice: Integration of Skype for Business with third-party audio conferencing providers has been retired. More details are here - https://docs.microsoft.com/skypeforbusiness/legal-and-regulatory/end-of-integration-with-3rd-party-providers. As a result, cmdlet no longer would work.
```

Reply:
Reading the more details url it says that the EoL was on July 31, 2021 so I would change "no longer would work" to "no longer works"